drydock/templates/kustomized/tutor13/extensions/prometheus-stack/kustomization.yml

```diff
@@ -0,0 +1,19 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: monitoring
+
+helmCharts:
+- name: kube-prometheus-stack
+  repo: https://prometheus-community.github.io/helm-charts
+  version: 38.0.2
+  releaseName: prometheus
+  includeCRDs: true
+{% if DRYDOCK_PROMETHEUS_VALUES|default(false) -%}
+  valuesInline: {{ DRYDOCK_PROMETHEUS_VALUES }}
+{% endif -%}
+
+{% if DRYDOCK_PROMETHEUS_OVERLAYS|default(false) -%}
+patches:
+{{ DRYDOCK_PROMETHEUS_OVERLAYS }}
+{% endif -%}
```

Comment:
when is this rendered? I run `tutor drydock save -r reference.yml` and nothing really happens with this nested files

Reply:
cc @MoisesGSalas do you know how this happens? could this work?

Reply:
Does this still work?
```
# kustomization.yml
patches:
['file']
```